deployments/setup-fork.ts

```diff
@@ -31,13 +31,14 @@ const {
 }: EnvOptions = process.env as any as EnvOptions;
 
 const tenderlyNetwork = tenderly.network();
+let forkId: string;
 
 const createTenderlyFork = async () => {
     console.log('Setting up new fork...');
 
     await tenderlyNetwork.initializeFork();
 
-    const forkId = tenderlyNetwork.getFork()!;
+    forkId = tenderlyNetwork.getFork()!;
```

Comment:
This change looks redundant.

Reply:
Why? It's setting the new global `forkId` variable